target_redshift/db_sync.py

```diff
@@ -500,19 +506,18 @@ def load_csv(self, s3_key, count, size_bytes, compression=False, manifest=False)
 
             full_columns.append(row)
 
-
         metrics = {}
 
         with self.open_connection() as connection:
             with connection.cursor(cursor_factory=psycopg2.extras.DictCursor) as cur:
                 inserts = 0
                 updates = 0
 
-                # Step 1: Create stage table if not exists
-                cur.execute(self.drop_table_query(is_stage=True))
-                sql = self.create_table_query(is_stage=True)
-                self.logger.info("Running create stage table: {}".format(sql))
-                cur.execute(sql)
+                run = partial(execute_sql, cur, self.logger)
```

Comment:
here is the run context creation. This gets rid of a few duplications

Reply:
yea code duplication. I feel harder to read code tends to be harder to debug and therefore harder to learn. So trying to make it easier to consume